.github/workflows/push-to-deploy.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: WordPress Plugin Deploy
-      uses: Nikschavan/action-wordpress-plugin-deploy@develop
+      uses: Nikschavan/action-wordpress-plugin-update@stable
```

Comment:
```suggestion
      uses: Nikschavan/action-wordpress-plugin-deploy@develop
```

Reply:
I think that's okay for Nikschavan repo, this repo is exist - https://github.com/Nikschavan/action-wordpress-plugin-deploy